### ISSUE TITLE
Missing `raw` in `Literal`(BigInt)

### ISSUE THREAD

**Issue by fisker** [facebook/flow#8962]
Flow version: 0.194.0

## Expected behavior

Should have `.raw` on `Literal`

## Actual behavior

```
require('flow-parser').parse('1n').body[0].expression
{
  type: 'Literal',
  loc: {
    source: null,
    start: { line: 1, column: 0 },
    end: { line: 1, column: 2 }
  },
  range: [ 0, 2 ],
  value: null,
  bigint: '1'   //<- Missing `raw`
}
```

* Link to Try-Flow or Github repo:

https://flow.org/try/#0IwOyA

**Comment by fisker** [comment]
This fix not included in 0.195.0?

**Comment by gkz** [comment]
@fisker, no, the next version - should be cut next week
